custom-namespaces.yaml

```
@@ -1774,11 +1753,26 @@ fxci:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.fxci_derived.task_run_costs_v1
```

Our standard in Looker is to add the user facing view -rather than the table.
```suggestion
        - table: moz-fx-data-shared-prod.fxci.task_run_costs
```